src/test/common/components/payment-intention/model/paymentPlan.ts

```diff
@@ -22,11 +23,12 @@ const DEFENDANT_PAYMENT_PLAN_FOR_DESERIALISATION = {
   instalmentAmount: 50,
   firstPaymentDate: { year: FUTURE_YEAR, month: 10, day: 10 },
   paymentSchedule: { value: PaymentSchedule.EVERY_MONTH.value, displayValue: PaymentSchedule.EVERY_MONTH.displayValue },
+  completionDate: MomentFactory.parse('2028-10-10'),
```

Comment:
Same comment as above expect that the payload simulate an object returned from the draft store.